### PR TITLE
GCE CR Support

### PR DESCRIPTION
**Pull request by jyejare**
$subject

### CODE REVIEW

**Comment by jyejare** [comment]
Was needed for https://github.com/SatelliteQE/robottelo/pull/7568 which is merged. Please help to review also the results are already submitted with the PR link in this comment.

File: nailgun/entities.py [hunk 1]
```diff
@@ -1295,6 +1295,22 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
...
 
 
+class GCEComputeResource(AbstractComputeResource):
+    """A representation of a Google Compute Resource entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'email': entity_fields.StringField(required=True),
+            'key_path': entity_fields.StringField(required=True),
+            'project': entity_fields.StringField(required=True),
```
Comment (ntkathole) [inline nailgun/entities.py]: apidoc shows email, key_path, project as optional... required=true needed for them?

Comment (jyejare) [reply nailgun/entities.py]: Yes, All of them are absolutely mandatory.

Comment (jyejare) [reply nailgun/entities.py]: Mandatory for GCE, but not for others. So probably that's the reason APIDoc says it's optional.

**Comment by mirekdlugosz** [comment]
I see OVirt and VMWare shadow `read` method to ignore password field - should GCE do something similar?

**Comment by jyejare** [comment]
@mirekdlugosz , There is no password in GCE, but there is key_path. And I dont find any harm exposing keypath stored under admins/root account. 

What do u think ?

**Pull request merged**